CardPayments/src/main/java/com/paypal/android/cardpayments/VaultListener.kt

```diff
@@ -0,0 +1,22 @@
+package com.paypal.android.cardpayments
+
+import androidx.annotation.MainThread
+import com.paypal.android.corepayments.PayPalSDKError
+
+/**
+ * Listener to receive callbacks form [CardClient.vault].
+ */
+interface VaultListener {
```

Comment:
Make internal to hide from public API until ready to ship this feature with docs?

Reply:
We could, but then we wouldn't be able to access the feature from the Demo app.

Reply:
Gotcha - is there a `nodoc` equivalent in Android where it's still technically accessible, just that the documentation is hidden?

Reply:
That's a good call. We can add `@suppress` for now that'll keep it from surfacing in public docs until we're ready to go for full GA.